dist/js/card.js

```diff
@@ -720,7 +720,7 @@ var render = function() {
   return _c("card", { staticClass: "px-4 py-4" }, [
     _c("div", { staticClass: "flex mb-4" }, [
       _c("h3", { staticClass: "mr-3 text-base text-80 font-bold" }, [
-        _vm._v("Most Visited Pages")
+        _vm._v(_vm._s(_vm.__("Most Visited Pages")))
```

Comment:
Do we have to do anything special to use `__()` in javascript, or does nova expose it for us?

Reply:
Nova does expos the `__()` in javascript. There are other configurations that might be required for other languages, but that happens on the app side, not the package side.
![Screen Shot 2021-04-13 at 4 08 07 PM](https://user-images.githubusercontent.com/7070136/114614505-99794b00-9c72-11eb-92c5-4c5fc77b0bb5.png)